src/parser.jl

```diff
@@ -1978,6 +1982,13 @@ function parse_resword(ps::ParseState)
         # export \n a      ==>  (export a)
         # export \$a, \$(a*b) ==> (export (\$ a) (\$ (parens (call-i a * b))))
         bump(ps, TRIVIA_FLAG)
+
+        # export scoped=true a, b ==> (export (= scoped true) b)
+        while peek(ps, 1) == K"Identifier" &&
+              peek(ps, 2) == K"="
+            parse_assignment(ps, x->parse_atsym(x, false))
```

Comment:
Maybe just write this out explicitly - if we want to pin the left hand side down to being a plain `K"Identifier"`, and not allow other things allowed by `parse_assignment` ... as we've already effectively done by peeking forward two symbols here:

```
p = position(ps)
bump(ps)
bump(ps)
parse_atom(ps)
emit(ps, K"=", p)
```

You probably want `parse_atom` on the right hand side of the `=` to allow general expressions; I feel that things like the following should probably parse (even if some of them are rejected by lowering):

```julia
x = :true
quote
    export scoped=$x A, B
end

export otherproperty=:foo otherthing=a+b A, B
```